### DISCUSSION TITLE
After importing photos, some many have title and keywords set to gibberish

### DISCUSSION THREAD

**Discussion by keyoke** [photoprism/photoprism#3978] [Q&A]
Hi,
Has anyone seen this before? any idea how to resolve it?

![image](https://github.com/photoprism/photoprism/assets/183547/6cc0b712-9d2e-4372-a821-d32cf6c13df8)

Regards
Gareth

**Comment by lastzero** [comment]
How do the original file names look like? I would think the titles are extracted from them unless your files include additional metadata.

**Reply by keyoke** [reply]
yes you are right! its a junk file name and looks like no meta data... need to work out how to clean up
